doc/1/guides/essentials/cli/index.md

```diff
@@ -431,6 +432,16 @@ The roles, profiles and users definition follow the same structure as in the bod
 }
 ```
 
+#### `--enable--plugins`
+<SinceBadge version="1.10.0" />
```

Comment:
Hmm... did we add new features in 1-dev recently? The next v1 version may be 1.9.3.

```suggestion
<SinceBadge version="1.10.0" />
```